Topics/Tech_Stacks/Mobile_Development/content.md

```diff
@@ -0,0 +1,140 @@
+# Introduction to Cross Platform vs Native Mobile Development
+![cpVnative](img1.png)
+
+## Table of Contents
+1. [Cross-Platform Development Frameworks](#cross-platform-development-frameworks)
+2. [Advantages and Challenges of Cross-Platform Development](#advantages-and-challenges-of-cross-platform-development)
+3. [Popular Cross-Platform Development Tools and Frameworks](#popular-cross-platform-development-tools-and-frameworks)
+4. [Native Mobile Development](#native-mobile-development)
+5. [Advantages and Challenges of Native Development](#advantages-and-challenges-of-native-development)
+6. [Comparing Performance and User Experience](#comparing-performance-and-user-experience)
+7. [Examples of Products](#examples-of-products)
+8. [Developer Resources](#developer-resources)
+9. [References](#references)
+
+## Cross-Platform Development Frameworks
+
+Cross-platform development for mobile involves creating applications that can run seamlessly across multiple operating systems, such as iOS and Android, utilizing a single codebase. Through cross-platform frameworks, developers gain the ability to streamline the development process by writing code once and deploying it across various platforms. CSC301 students can benefit greatly from understanding cross-platform development as it is easier to implement for multiple famous platforms such as iOS and Android. Students with existing knowledge in web can easily start-off with React Native providing them with a clear transition.
+
+## Advantages and Challenges of Cross-Platform Development
+
+### Advantages
+
+1. Faster development and deployment times compared to native development
+2. Cost-effectiveness due to code reuse across multiple platforms
+3. Simplified maintenance and updates with a single codebase
+4. Greater reach by targeting multiple platforms simultaneously
+
+### Challenges
+
+1. Performance limitations compared to native development
+2. Platform-specific nuances and limitations may require workarounds
+3. Dependency on third-party frameworks and libraries, which may introduce compatibility issues
+4. Limited access to platform-specific features and APIs may restrict functionality
+
+## Popular Cross-Platform Development Tools and Frameworks
```

Comment:
Include a brief mention of hybrid app development approaches, such as using web views within native apps, to provide a more comprehensive overview of cross-platform options.

Reply:
Noted